app/_data/docs_nav_mesh_2.1.x.yml

```diff
@@ -425,245 +415,3 @@ items:
       - text: Contribute to Mesh
         url: /community/contribute-to-mesh/
         src: /.repos/kuma/app/_src/community/contribute-to-kuma/
-
-
-unlisted:
```

Comment:
@lahabana just to confirm, we no longer want to generate these pages, is that correct?

Reply:
Yes there are not bringing much